install_data.sh

```diff
@@ -215,7 +215,7 @@ install_files() {
   data+=("5fddbc109c82980f9436aa5c21a57c61" "resources/GRCh38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz.fai" "")
   data+=("db66bd01c2cb8a1ccb81c486239fa616" "resources/GRCh38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz.gzi" "")
   data+=("798b74ca2ff85b976ab51aab3f515c69" "resources/GRCh38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.gz.mmi" "")
-  data+=("c8c50252f1874ce3e7029b2178b2991a" "resources/GRCh38/GCF_000001405.39_GRCh38.p13_genomic_mapped.gff.gz" "")
+  data+=("5390c5802c3ed16bde9ab4d6047447f5" "resources/GRCh38/GCF_000001405.40_GRCh38.p14_genomic_mapped.gff.gz" "")
```

Comment:
to stay in sync `GCA_000001405.15/GCF_000001405.26` should be used instead of `GCA_000001405.29/GCF_000001405.40`?